Docs for initial release of Forms 14 (#6118)
umbraco/UmbracoDocs: 14/umbraco-forms/developer/extending/adding-a-fieldtype.md

- Recordset actions - `formRecordSetActions`
- Workflows - `formProviderWorkflows`

The `View` property indicates a property editor UI used for editing the setting value. You can use a built-in property editor UI, one from a package, or a custom one registered with your solution.  The default value if not provided is `Umb.PropertyEditorUi.TextBox`, which will use the standard Umbraco text box property editor UI.

> You may optionally want to register a settings value converter. This is a client-side, manifest registered component, that converts between the setting value required for the editor and that persisted with the form definition. A converter defines three methods:

Reviewer: ```suggestion
You may want to consider registering a settings value converter. This is a client-side component that is registered in the manifest. It converts the setting value required for the editor and the value persisted with the form definition. A converter defines three methods:
```

Author: Improves on what I had for sure, but to be precise I think should be:

```
You may want to consider registering a settings value converter. This is a client-side component that is registered in a manifest. It converts between the setting value required for the editor and the value persisted with the form definition. A converter defines three methods:
```

Reviewer: Thanks, I'll make the change then 💪

> You may optionally want to register a settings value converter. This is a client-side, manifest registered component, that converts between the setting value required for the editor and that persisted with the form definition. A converter defines three methods:

Reviewer: ```suggestion
You may want to consider registering a settings value converter. This is a client-side component that is registered in a manifest. It converts between the setting value required for the editor and the value persisted with the form definition. A converter defines three methods:
```
